src/static/bitbucket/components/App.tsx

```diff
@@ -31,7 +31,7 @@ export class App extends React.Component {
 
   messageEmoji = {
     default: '📢',
-    warning: '⚠',
+    warning: '⚠️',
```

Comment:
?

Reply:
I've got no idea what happened here, I think I accidentally changed it in a previous PR. It looks the same but has a different unicode I think.